dataprofiler/data_readers/json_data.py

```diff
@@ -63,21 +64,93 @@ def __init__(self, input_file_path=None, data=None, options=None):
     def selected_keys(self):
         return self._selected_keys
 
+    def _find_data(self, json_lines, path=""):
+        """
+        Finds all the column headers/data in a Json and returns them as a 
+        list.
+
+        :param json_lines: json format list of dicts or dict
+        :type json_lines: list or string or dict
+        :return: list of dicts of {column headers: values}
+        """
+        if path != "":
+            path = path + "."
+
+        list_of_dict = []
+        if isinstance(json_lines, dict):
+            for key in json_lines:
+                if isinstance(json_lines[key], dict): # If nested dictionary
+                    list_of_dict = list_of_dict + self._find_data(
+                        json_lines[key], path + key)
+                elif isinstance(json_lines[key], list): # If list
+                    if isinstance(json_lines[key][0], dict): # If list of dictionaries
```

Comment:
Should probably have check to make sure its not an empty list, otherwise this index will throw an error.